examples/rfcavity/run_rfcavity_ref_part.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2023 ImpactX contributors
+# Authors: Marco Garten, Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+# -*- coding: utf-8 -*-
+
+from impactx import ImpactX, distribution, elements
+
+sim = ImpactX()
+
+# set numerical parameters and IO control
+sim.space_charge = False
+# sim.diagnostics = False  # benchmarking
+sim.slice_step_diagnostics = True
+
+# domain decomposition & space charge mesh
+sim.init_grids()
+
+# load a 230 MeV electron beam with an initial
+# unnormalized rms emittance of 1 mm-mrad in all
+# three phase planes
+kin_energy_MeV = 230.0  # reference energy
+bunch_charge_C = 1.0e-10  # used with space charge
+npart = 10000  # number of macro particles (outside tests, use 1e5 or more)
+
+#   reference particle
+ref = sim.particle_container().ref_particle()
+ref.set_charge_qe(-1.0).set_mass_MeV(0.510998950).set_kin_energy_MeV(kin_energy_MeV)
+
+#   particle bunch
+distr = distribution.Waterbag(
+    lambdaX=0.352498964601e-3,
+    lambdaY=0.207443478142e-3,
+    lambdaT=0.70399950746e-4,
+    lambdaPx=5.161852770e-6,
+    lambdaPy=9.163582894e-6,
+    lambdaPt=0.260528852031e-3,
+    muxpx=0.5712386101751441,
+    muypy=-0.514495755427526,
+    mutpt=-5.05773e-10,
+)
+sim.add_particles(bunch_charge_C, distr, npart)
+
```

Comment:
Does this work if we skip the particle init (and only init the ref particle in the lines above)?
```suggestion
```